### DISCUSSION TITLE
code sample do not have imports

### DISCUSSION THREAD

**Discussion by iceh2o** [xrutayisire/react-js-cron#24] [General]
Hi, I am now to react.  First time using your react-js-cron package and got lost a bit.  I read the documentation a couple of times and tried with the demo and copied the code to my react app.  However, the UI doesn't show up or it's showed up in a messed up way.  Can you provide a complete demo code including the import statements?

**Comment by xrutayisire** [comment]
Hello,

Code sample can be found directly in the stories :)

File: https://github.com/xrutayisire/react-js-cron/blob/master/src/stories/index.stories.tsx

UI view:  <img width="1680" alt="Screenshot 2022-04-15 at 11 09 41" src="https://user-images.githubusercontent.com/19946868/163551624-0bfa5a8d-bf3c-48d5-810d-19e56d5e97d6.png">

**Comment by iceh2o** [comment]
Perfect! Thank you so much.

**Reply by iceh2o** [reply]
I was following the sample from index.stores.tsx but I wasn't able to import the useCronReducer.

import { useCronReducer } from './utils.stories'

I am not able to find the utils.stores on the npm installed package (node_modules folder).  

Below is a listing of files I see under the react-js-cron folder
04/14/2022  02:05 PM    <DIR>          components
04/14/2022  02:05 PM             1,346 constants.ts
04/14/2022  02:05 PM            12,798 converter.ts
04/14/2022  02:05 PM            10,046 Cron.tsx
04/14/2022  02:05 PM    <DIR>          fields
04/14/2022  02:05 PM               250 index.ts
04/14/2022  02:05 PM             1,791 locale.ts
04/14/2022  02:05 PM             3,694 styles.css
04/14/2022  02:05 PM             8,307 types.ts
04/14/2022  02:05 PM             1,495 utils.ts

**Reply by xrutayisire** [reply]
Yes, stories are not supposed to be published on the npm package. Stories are just documentation. The example is based on a custom hook I created and available here: https://github.com/xrutayisire/react-js-cron/blob/master/src/stories/utils.stories.ts

The way you are using the Cron component is up to you, and you can use the same reducer if you want but also manage the values another way.

**Reply by iceh2o** [reply]
thanks for the quick reply.  I understand I can copy and use the same reducer in my code.
